ShareX.HelpersLib/Helpers/ImageHelpers.cs

```diff
@@ -2022,5 +2024,123 @@ public static Size GetImageFileDimensions(string filePath)
 
             return Size.Empty;
         }
+
+        public static bool StripImageMetadata(string fileName)
+        {
+            bool result = false;
+
+            try
+            {
+                string tempName = Path.Combine(Path.GetDirectoryName(fileName), Guid.NewGuid().ToString());
+                using (FileStream stream = new FileStream(fileName, FileMode.Open, FileAccess.ReadWrite))
+                {
+                    BitmapDecoder original = BitmapDecoder.Create(stream, BitmapCreateOptions.PreservePixelFormat, BitmapCacheOption.None);
+                    BitmapEncoder output = null;
+                    string extension = Path.GetExtension(fileName);
+                    switch (extension)
+                    {
+                        case ".png":
+                            output = new PngBitmapEncoder();
+                            break;
+                        case ".jpeg":
+                        case ".jpg":
+                            output = new JpegBitmapEncoder();
+                            break;
+                        case ".tif":
+                            output = new TiffBitmapEncoder();
+                            break;
+                    }
+
+                    if (original.Frames[0]?.Metadata != null)
+                    {
+                        BitmapFrame copyFrame = (BitmapFrame)original.Frames[0].Clone();
+                        BitmapMetadata metaData = original.Frames[0].Metadata.Clone() as BitmapMetadata;
+
```

Comment:
These spaces that are not after any ``}`` seems unnecessary for me. could remove these too.

Also helps with keeping things short and readable.